samples/azure-quantum/qae-numerical-integration/QuantumAmplitudeEstimation.ipynb

```diff
@@ -0,0 +1,1148 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_average_err(k_err_avg)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In our graph we see our results are not as on the simulator. \n",
+    "In fact, in our testing $k=0$ performed better than any other $k$.\n",
+    "This comes down to the noise effects of the additional gates we need for the larger values of $k$.\n",
+    "This is an interesting result because it shows considerations for current quantum computers that a perfect simulator does not depict: Every gate introduces noise.\n",
+    "So one of our goals must be to reduce circuit depth. \n",
+    "Despite having the reduced depth already using the spin echo optimization, we still have noise effects. \n",
+    "Without this optimization's circuit depth reduction we'd expect noise to be even higher.\n"
```

Comment:
Ending on something negative feels like it undercuts the goal of the sample: try to end on a positive note with calls to action for the reader to continue using quantum hardware.